packages/data-service/src/data-service.ts

```diff
@@ -79,18 +79,20 @@ import type { CSFLECollectionTracker } from './csfle-collection-tracker';
 import { CSFLECollectionTrackerImpl } from './csfle-collection-tracker';
 
 import * as mongodb from 'mongodb';
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-ignore no types for 'extension' available
-import { extension } from 'mongodb-client-encryption';
-import type {
-  ClientEncryption as ClientEncryptionType,
-  ClientEncryptionDataKeyProvider,
-  ClientEncryptionCreateDataKeyProviderOptions,
-} from 'mongodb-client-encryption';
-// mongodb-client-encryption only works properly in a packaged
-// environment with dependency injection
-const ClientEncryption: typeof ClientEncryptionType =
-  extension(mongodb).ClientEncryption;
+
+// TODO: replace any with proper types when the Node bundles native binding distributions
+// https://jira.mongodb.org/browse/WRITING-10274
+let ClientEncryption: any;
```

Comment:
This one can actually be properly typed, right?

```suggestion
import type { ClientEncryption as ClientEncryptionType } from 'mongodb-client-encryption';
let ClientEncryption: ClientEncryptionType;
```